Cython/Compiler/Builtin.py

```diff
@@ -363,6 +363,20 @@ def declare_in_type(self, self_type):
         # TODO - format would be nice, but hard to get
         # __len__ can be accessed through a direct lookup of the buffer (but probably in Optimize.c)
         # error checking would ideally be limited api only
+        BuiltinProperty("buf", PyrexTypes.c_void_ptr_type, '__Pyx_PyMemoryView_Get_buf',
+                        exception_value=0, exception_check=True,
+                        utility_code=TempitaUtilityCode.load_cached(
+                            "memoryview_get_from_buffer", "Builtins.c",
+                            context=dict(name="buf")
+                        )
+        ),
```

Comment:
IIUC we would rather drop this as it is not in the Python implementation. Though please let me know if I've misunderstood

```suggestion
```

Reply:
Yes - my view is drop this